FirebaseCore/Sources/Public/FirebaseCore/FIROptions.h

```diff
@@ -99,25 +99,29 @@ NS_SWIFT_NAME(FirebaseOptions)
 @property(nonatomic, copy, nullable) NSString *appGroupID;
 
 /**
- * Initializes a customized instance of FIROptions from the file at the given plist file path. This
- * will read the file synchronously from disk.
- * For example,
- * NSString *filePath =
- *     [[NSBundle mainBundle] pathForResource:@"GoogleService-Info" ofType:@"plist"];
- * FIROptions *options = [[FIROptions alloc] initWithContentsOfFile:filePath];
+ * Initializes a customized instance of FirebaseOptions from the file at the given plist file path.
+ * This will read the file synchronously from disk.
+ * For example:
+ * ```swift
+ *   if let path = Bundle.main.path(forResource:"GoogleServices-Info", ofType:"plist") {
+ *       let options = FirebaseOptions.init(contentsOfFile: path)
```

Comment:
Nit: prefer `FirebaseOptions(contentsOfFile: path)` instead of explicitly calling `.init`